cli/src/factory_ops/boxing/init

```diff
@@ -95,6 +95,17 @@ fi
 # Prepare shutdown
 ensure_healthy_cluster
 
+echo "INFO: Resolving service user credentials" | tee -a ${LOG_FILE}
+# FIXME these values might be encrypted
+service_user_name=$(salt-call pillar.get system:service_user:name --output=newline_values_only)
+service_user_secret=$(salt-call pillar.get system:service_user:password --output=newline_values_only)
```

Comment:
same here, use --local option

Reply:
I copied from [here](https://github.com/Seagate/cortx-prvsnr/blob/e6d15babe156505f810b912ccd0918112cc42788/cli/src/factory_ops/boxing/prov_tasks#L76). What is the difference?

Reply:
yes, not needed for pillar.get commands. Ignore.